packages/ses/src/get-anonymous-intrinsics.js

```diff
@@ -135,6 +119,27 @@ export const getAnonymousIntrinsics = () => {
     '%InertCompartment%': InertCompartment,
   };
 
+  if (AsyncGeneratorFunctionInstance !== undefined) {
```

Comment:
Please note that this block is conditional because Hermes does not provide these intrinsics.

Reply:
Is it possible for `%AsyncGeneratorPrototype%` to be absent while `%AsyncIteratorPrototype%` is present? If so, is there some means to discover `%AsyncIteratorPrototype%` without `%AsyncGeneratorPrototype%`?

I see the code below assumes that there might be a `globalThis.AsyncIterator` constructor. It seems possible for this to exist in a system without `%AsyncGeneratorPrototype%`, providing a distinct means of discovering `%AsyncIteratorPrototype%`.

Reply:
> Is it possible for `%AsyncGeneratorPrototype%` to be absent while `%AsyncIteratorPrototype%` is present?

yes ^ when a Readable Stream is created, it's possible to get an AsyncIterator from it

> If so, is there some means to discover `%AsyncIteratorPrototype%` without `%AsyncGeneratorPrototype%`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AsyncIterator#description

> I see the code below assumes that there might be a `globalThis.AsyncIterator` constructor. It seems possible for this to exist in a system without `%AsyncGeneratorPrototype%`, providing a distinct means of discovering `%AsyncIteratorPrototype%`.

one way we know of is Readable Stream